api/spaces/spaces.go

```diff
@@ -31,27 +31,18 @@ func NewAPI(caller base.APICallCloser) *API {
 	}
 }
 
-func makeCreateSpaceParams(name string, subnetIds []string, public bool) params.CreateSpaceParams {
-	spaceTag := names.NewSpaceTag(name).String()
-	subnetTags := make([]string, len(subnetIds))
-
-	for i, s := range subnetIds {
-		subnetTags[i] = names.NewSubnetTag(s).String()
-	}
-
-	return params.CreateSpaceParams{
-		SpaceTag:   spaceTag,
-		SubnetTags: subnetTags,
-		Public:     public,
-	}
-}
-
 // CreateSpace creates a new Juju network space, associating the
 // specified subnets with it (optional; can be empty).
 func (api *API) CreateSpace(name string, subnetIds []string, public bool) error {
```

Comment:
Shall we rename the `subnetIds` argument?

Reply:
yes, missed this one.  ty